src/commands/new/git.rs

```diff
@@ -0,0 +1,20 @@
+use git2::{Error, Repository, RepositoryInitOptions};
```

Comment:
this shouldn't be in commands/new/git. otherwise we would expect there to be `aqora new git`. Just put it in the `src` for now. At some point when things get too messy we'll do a little reorganization. But also this should apply to `aqora template` as well as `aqora new`